README.md

```diff
@@ -1,315 +1,277 @@
+
+     ```yaml
+     spring:
+       cache:
+         type: none
+     ```
+
+   See [here](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-caching.html#boot-features-caching-provider-none)
+   for details.
+
+4. Configure Keycloak and Google SSO
+
+   To log in to your app, you'll need to have [Keycloak](https://keycloak.org)
+   up and running.
+
+   ```sh
+   docker-compose -f src/main/docker/keycloak.yml up
+   ```
+
+   1. Once keycloak server is running, go to `http://localhost:8080` and click
+      `Administration Console`
+   2. Login with the credential `username: admin, password:admin`.
+
+      - Note this will match the credentials in the [keycloak.yml](src/main/docker/keycloak.yml).
+
+   3. [Create a realm](https://www.keycloak.org/docs/latest/server_admin/#proc-creating-a-realm_server_administration_guide)
+      and name it `oncokb-curation`
+   4. [Create a new openid-connect client](https://www.keycloak.org/docs/latest/server_admin/#assembly-managing-clients_server_administration_guide)
+      - Set Access Type to `confidential`
+      - Enable Standard Flow
+      - Enable implicit flow
+      - Disable access grants
+      - Disable service accounts
+      - Disable OAuth 2.0 Device Authorization Grant
+      - Disable Authorization
+      - Set valid Redirect URI's to `http://localhost:*` and `https://localhost:*`
+      - Set Web Origins to `*`
+   5. Follow [instructions](https://support.google.com/cloud/answer/6158849)
+      to obtain Google Oauth2 client id and secret.
+   6. In Keycloak, go to **Identity Providers** >
+      **Edit button on google provider** > **Replace client id and secret**
+   7. Copy the redirect URL for the google identity provider.
+      It should look like `http://localhost:8080/realms/oncokb-curation/broker/google/endpoint`
+   8. Go back to Google APIs & Services `Credentials` tab and click on your
+      application. Add the redirect URL from the last step into the list with
+      the label `Authorized redirect URIs`.
+
+   9. Update the security settings in [application-dev.yml](src/main/resources/config/application-dev.yml).
+
+      - Include realm name in the `spring.security.oauth2.client.provider.oidc.issue-uri`
+        property.
+      - To find the client secret, go to keycloak admin console and look for `web_app`
+        client. Find the `Credentials` tab and copy the client secret.
+        Credentials tab will only show when the client's `Access Type = confidential`.
+
+      ```yaml
+      spring:
+        ...
+        security:
+          oauth2:
+            client:
+              provider:
+                oidc:
+                   issuer-uri: http://localhost:8080/realms/oncokb-curation
+              registration:
+                oidc:
+                  client-id: web_app
+                  client-secret: CLIENT_SECRET
+                  scope: openid,profile,email
+      ```
+
+5. Configure Firebase
```

Comment:
@zhx828 I made major changes to the firebase section so let me know if it still looks good.